docs/pages/guides/server-components.mdx

```diff
@@ -68,9 +64,9 @@ export default function Index() {
 
 ## Layout Routes
 
-> Complex built-in layouts such as Stack, and Tabs are not supported yet. They'll fallback to a `<Slot />`.
+> Complex built-in layouts such as Stack and Tabs are not yet supported. They will fallback to a `<Slot />`.
 
-Layout routes (`app/_layout.js`) can be used to wrap all routes in a common layout.
+Layout routes (**app/_layout.tsx**) can wrap all routes in a common layout.
```

Comment:
```suggestion
Layout routes (**app/\_layout.tsx**) can wrap all routes in a common layout.
```

My bad, the linter below is right. This is the correct markdown formatting for using underscore in before a file name here.